src/rust/bitbox02-rust/src/workflow/confirm.rs

```diff
@@ -16,13 +16,19 @@ use crate::bb02_async::option;
 
 pub use bitbox02::ui::{ConfirmParams as Params, Font};
 
+pub struct UserAbort;
```

Comment:
I wonder if if makes sense to start defining common errors. I imagine something like this in `bitbox02::ui` mod:

```rust
enum ErrorKind {
  UserAbort,
  IncorrectPassword,
  InvalidInput,
}
```

Then can use this error enum in all UI and workflow related modules. I think it would work well for `UserAbort` and reduce code for `core::convert::From` in API conversion, but OTOH I like reading `UnlockError::IncorrectPassword` because it tells me exactly where it's coming from. So, not sure but maybe something to consider in the future.